src/pages/OfferNew/Perils/InsuranceValues/index.tsx

```diff
@@ -115,6 +118,14 @@ export const InsuranceValues: React.FC<Props> = ({ insuranceType }) => {
           <DocumentIcon />
           {textKeys.COVERAGE_PRESALEINFORMATION_BUTTON()}
         </Link>
+
+        <Link
+          href={textKeys[getEUPrebuyPDFTextKey(insuranceType)]()}
+          target="_blank"
+        >
```

Comment:
Shall we maybe hide this if the text key doesn't exist? considering we don't have it in english?

Reply:
True! Just a heads up, the existing pre-sale information on the english page links to a pdf in Swedish: https://cdn.hedvig.com/info/forkopsinformation-bostadsratt-2018-02.pdf